sdk/android/.github/workflows/github-actions-concept.yml

```diff
@@ -0,0 +1,61 @@
+name: GitHub Actions Concept
+run-name: Android instrumentation tests proof of concept
+on: [push]
+
+jobs:
+  connected-tests:
+    runs-on: macos-latest
```

Comment:
Why does this need to run on macos? Probably best to run on Linux?

Reply:
There's an explanation of this choice [here](https://github.com/ReactiveCircus/android-emulator-runner/issues/46), long story short it's a probelm with the GitHub actions Linux VMs